app/views/quotes/index.html.erb

```diff
@@ -1,3 +1,6 @@
 <% @quotes.each do |quote| %>
-  <%= render quote %>
+  <div class="well">
+    <%= link_to "->", quote %>
+    <%= render quote, vote: quote.votes.find_by(user: current_user) %>
```

Comment:
I am a bit torn on this one, in general I prefer passing locals to partials because it makes it clear what you need to render them. In this case however the vote is actually not needed directly by the quote partial, but by the voting partial which is rendered inside the quote partial, so I would suggest moving this logic to get the vote there. The benefit of that would be that if you ever change the quote partial to no longer show the vote, you don't need to change it's public API.